README.md

```diff
@@ -117,10 +115,16 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc
+$ ./x.py dist --build=x86_64-pc-windows-msvc --install
```

Comment:
@keeperofdakeys Kinda funny that the only section on msys2 simply tells you to target msvc, despite rustbuild defaulting to gnu if it detects an msys environment. Maybe we could replace this section with a thing that describes this default target behavior based on whether you're in an msys environment (really whether you have `uname` in your path), and how you can use `config.toml` to override the default.